testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -3717,6 +3739,222 @@ fn cost_voting_integration() {
     channel.stop_chains_coordinator();
 }
 
+#[test]
+#[ignore]
+fn mining_events_integration_test() {
```

Comment:
Can you add this test to the `sampled-genesis` test matrix in `.github/workflows/bitcoin-tests.yml`?